log-analyzer/features/org.wso2.analytics.common.la.apim.feature/src/main/resources/gadgets/LogErrorMessage/js/main.js

```diff
@@ -134,56 +134,56 @@ function fetch() {
         receivedData = JSON.parse(d["message"]);
         if (d["status"] === "success") {
             tomorrow = new Date(from);
-            if(timeFrame==="daily"){
+            if (timeFrame === "daily") {
                 newFrom.setHours(0);
                 newFrom.setMinutes(0);
                 newFrom.setSeconds(0);
                 newTo.setHours(0);
                 newTo.setMinutes(0);
                 newTo.setSeconds(0);
-                while(!(newFrom.getTime() >= newTo.getTime())){
-                    mockData.push([newFrom.toDateString(),0,"No entries","No entries",0]);
-                    newFrom.setHours(newFrom.getHours()+24);
+                while (!(newFrom.getTime() >= newTo.getTime())) {
```

Comment:
Hi Tharik,

This is fixed by the commits in : https://github.com/wso2/shared-analytics/pull/90

Thanks,
SajithD